semgrep/semgrep/meta.py

```diff
@@ -31,72 +26,109 @@ class GitMeta:
     cli_baseline_ref: Optional[str] = None
     environment: str = field(default="git", init=False)
 
-    @cachedproperty
+    @property
```

Comment:
None of these variables are called more than once so no need to cache (there are like 2 that are called but they arent slow so w/e)